ext/mysqli/tests/mysqli_rollback.phpt

```diff
@@ -53,6 +53,13 @@ if (!have_innodb($link))
         echo $exception->getMessage() . "\n";
     }
 
+    try {
+        $link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket, false);
```

Comment:
in all 3 tests this connection create statement should be put before the try statement